stable/yugaware/values.yaml

```diff
@@ -11,7 +10,10 @@ image:
   repository: quay.io/yugabyte/yugaware
   tag: 2.5.1.0-b153
   pullPolicy: IfNotPresent
-  pullSecret: yugabyte-k8s-pull-secret
+  pullSecret: yugabyte-k8s-pull-secret-1
```

Comment:
We should not change the default name here. If someone has existing deployment, and they upgrade, image pull will fail in this case.

Reply:
Good catch!